cmd/up/up.go

```diff
@@ -617,16 +617,12 @@ func (up *upContext) start() error {
 	pidFileCh := make(chan error, 1)
 
 	analytics.TrackUp(analytics.TrackUpMetadata{
-		IsInteractive:          up.getInteractive(),
+		Mode:                   up.Dev.GetMode(),
 		IsOktetoRepository:     utils.IsOktetoRepo(),
-		IsV2:                   up.Manifest.IsV2,
```

Comment:
I'd keep the v2 parameter, since older version of the CLI are going to keep sending it, we will not have an easy way to distinguish old ones and new ones without it. I understand we are trying to make it compliant with the spec, but even if this was not added there I would keep it just in case it affects our dashboards in mixpaneI. Is there any other reason why we are removing it?

So I gues what I'm trying to say is to add all those that are defined in the dictionary, and at the end keep old those that were already there for retro-compatibility.

Reply:
i've updated, so i kept the old ones and added the new ones in both exec and up